### ISSUE TITLE
REACT: Not all constructs are visible inside conditions

### ISSUE THREAD

**Issue by HayleyMills** [CLOSER-Cohorts/archivist#513]
True and else branches have been added to the construct page but sometimes have constructs inside them and sometimes they don't- I'm not sure what the reason for this is? This is an example from https://closer-react.herokuapp.com/instruments/rat_2/build/constructs/
![image](https://user-images.githubusercontent.com/16664703/106638026-4e103580-657b-11eb-8cd9-f5b1935b0b68.png)

Also see https://closer-react.herokuapp.com/instruments/ncds_91_cmi/build/constructs/
![image](https://user-images.githubusercontent.com/16664703/106638275-9c253900-657b-11eb-9ad6-c3df22fa3e4c.png)

**Comment by simonreed** [comment]
I can't seem to see any conditions that should have true and false children on those instruments on the staging site either. If I create a new construct so that they have both true and false children then they appear. Is this potentially just an issue with data we've got on staging?

https://closer-archivist-staging.herokuapp.com/instruments/ncds_91_cmi/build/constructs
https://closer-archivist-staging.herokuapp.com/instruments/rat_2/build/constructs

**Comment by HayleyMills** [comment]
I think they may have been a misunderstanding, whilst there are occasions when there is a true and and else branch which have constructs inside both, this is not very common. 

The main current problem, is that not all the constructs are appearing in the conditions. e.g. This doesn't have any constructs inside c_qA1b in the REACT version. 
![image](https://user-images.githubusercontent.com/16664703/106892100-829f0100-66e3-11eb-9158-f8debaf325c2.png)
https://closer-react.herokuapp.com/instruments/ncds_91_cmi/build/constructs/

But you can see there are lots of constructs on the staging sever version..
![image](https://user-images.githubusercontent.com/16664703/106892035-6c914080-66e3-11eb-9045-dd3ce6cf5a26.png)
https://closer-archivist-staging.herokuapp.com/instruments/ncds_91_cmi/build/constructs

**Comment by simonreed** [comment]
@HayleyMills had you previously look at this instrument before we added in the conditions into the constructs? Sometimes if you reordered an instrument before we added the conditions it caused issues with the conditions losing some entries. 

My version of that instrument is working as expected so I wonder if the reorder of the data being wrongly updated is the issue here. @spuddybike would it be possible to recreate the heroku-react database from a copy of the staging database so that we can start with a clean db again?
